content/en/examples/pods/security/seccomp/fields.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod
+spec:
+  securityContext:
+    seccompProfile:
+      type: Unconfined
+  ephemeralContainers:
+  - name: ephemeral-container
+    image: debian
+    securityContext:
+      seccompProfile:
+        type: RuntimeDefault
+  containers:
+  - name: container
+    image: debian
```

We like to encourage good practise:
```suggestion
    image: docker.io/library/debian:stable
```

---

Thanks, fixed :+1: